lisa/features/security_profile.py

```diff
@@ -16,13 +16,18 @@
 @dataclass()
 class SecurityProfileSettings(schema.FeatureSettings):
     type: str = FEATURE_NAME_SECURITY_PROFILE
+    security_type: str = ""
```

Comment:
Can we use only one enum for the three fields? The default value is "disabled", and it can be "boot" or "full". The string is easy to make mistake.

Reply:
I confirmed with Johnson that CVM validation does not require the ability to boot CVM without secure boot. I also don't think there are any cases where we would need TrustedLaunch without vTpm or secureBoot. Therefore, I have reduced the three fields to one enum with the options "none" (default), "boot", and "cvm". This also removes the possibility for runbook and test requirements to conflict (as you mentioned below).